lib/awsv4/core.js

```diff
@@ -108,16 +107,17 @@ function createCanonicalHeaders(headers) {
         "\n"
       );
     })
+    .sort()
```

Comment:
We need to move the `sort()` to occur after the `toLowerCase()` call in the block, as the case change modifies the sorting outcome.